integration/topic.sh

```diff
@@ -41,7 +37,9 @@ $ALLORAD_BIN tx emissions push-topic \
   "$PT_INFERENCE_METHOD" \
   "$PT_INFERENCE_CADENCE" \
   "$PT_DEFAULT_ARG" \
-  --yes --keyring-backend=test --chain-id=demo;
+  --yes --keyring-backend=test --chain-id=demo \
+  --gas-prices=1uallo --gas=auto --gas-adjustment=1.5;
```

Comment:
It's optional but you can remove this since we removed the gas fees again

Reply:
i'm going to keep them, makes the tests less likely to arbitrarily fail is my feeling